types/index.esm.d.ts

```diff
@@ -70,6 +70,7 @@ declare module 'chart.js' {
       datasetOptions: SankeyControllerDatasetOptions;
       defaultDataPoint: SankeyDataPoint;
       parsedDataType: SankeyParsedData;
+      metaExtensions:{}
```

Comment:
Does not look like anything in this repository. What is the error you are trying to fix?

Reply:
Nvm, this is what was changed in Chart.js.